iam/iam-gcp/src/test/java/com/salesforce/multicloudj/iam/gcp/GcpIamIT.java

```diff
@@ -0,0 +1,186 @@
+package com.salesforce.multicloudj.iam.gcp;
+
+import com.google.api.gax.core.FixedCredentialsProvider;
+import com.google.api.gax.rpc.TransportChannelProvider;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.iam.admin.v1.IAMClient;
+import com.google.cloud.iam.admin.v1.IAMSettings;
+import com.salesforce.multicloudj.common.gcp.GcpConstants;
+import com.salesforce.multicloudj.common.gcp.util.MockGoogleCredentialsFactory;
+import com.salesforce.multicloudj.common.gcp.util.TestsUtilGcp;
+import com.salesforce.multicloudj.iam.client.AbstractIamIT;
+import com.salesforce.multicloudj.iam.client.IamClient;
+import com.salesforce.multicloudj.iam.client.TestIamClient;
+import com.salesforce.multicloudj.iam.driver.AbstractIam;
+import org.junit.jupiter.api.Assertions;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Integration tests for GcpIam.
+ * 
+ * <p>This test class follows the conformance testing pattern used across the MultiCloudJ project.
+ * 
+ * <p><b>Important Note:</b> GCP IAM Admin API uses gRPC protocol, which WireMock does not
+ * natively support. Therefore:
+ * <ul>
+ *   <li>Recording mode (-Drecord) works by connecting directly to real GCP APIs</li>
+ *   <li>Replay mode is not yet implemented and will fail</li>
```

Comment:
we should try the rest client until we have the grpc enabled and avoid skipping the tests.
grpc is supported by wiremock here https://github.com/wiremock/wiremock-grpc-extension